.travis.yml

```diff
@@ -1,8 +1,19 @@
-language: java
+matrix:
+  include:
+    - language: java
+      jdk: oraclejdk8
 
-# Only 8 is suitable
-jdk:
-  - oraclejdk8
-
-# Default maven settings and testing is enough, currently.
+    - language: node_js
+      node_js:
+        - '8'
+      branches:
+        only:
+          - master
+      script:
+        - git config --global user.name "${GH_NAME}"
+        - git config --global user.email "${GH_EMAIL}"
+        - echo "machine github.com login ${GH_NAME} password ${GH_TOKEN}" > ~/.netrc
+        - cd docs/site && npm install && GIT_USER="${GH_USER}" npm run publish-gh-pages
```

Comment:
@allenlee820202 it should successfully deploy docs to gh-page via `npm run publish-gh-pages` ... ([doc](
https://docusaurus.io/docs/en/publishing#using-travis-ci)).

You can also use `deploy` phase in travis with manual scripts.